index.bs

```diff
@@ -770,8 +771,9 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
               <li>
                 Set |videoSupported| to the result of running
                 [$check MIME type support$] with
-                |videoMimeType| and
-                |configuration|'s {{MediaEncodingConfiguration/type}}.
+                |videoMimeType|,
+                |configuration|'s {{MediaEncodingConfiguration/type}},
+                and <code>undefined</code> for |colorGamut| and |transferFunction|.
```

Comment:
It's better to not pass in anything, as it's literally "undefined". 

```suggestion
                |configuration|'s {{MediaEncodingConfiguration/type}}.
```

Reply:
Done.